tests/test_pycita.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env python
+
+"""测试CitaClient."""
+from pathlib import Path
+import json
+import pytest
+
+from pycita import CitaClient, ContractClass
+from pycita import equal_param, encode_param, decode_param, join_param, param_to_bytes, param_to_str
+
+# uncomment下述代码, 会开始输出日志
+# import logging
+# logging.basicConfig(level=logging.INFO)
+# 使用了SimpleStorage.sol作为样本
+
+client = CitaClient('http://192.168.127.216:1338')
```

Comment:
should use `http://127.0.0.1:1337` by default, or use a environment variable so that we can change the value at run time.

Reply:
Fixed. But maybe need a valid url for Travis CI.